source/whats-new.txt

```diff
@@ -35,6 +36,26 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.4:
+
+What's New in 5.4
+-----------------
+
+New features of the 5.4 {+driver-short+} release include:
+
+.. important:: Deprecation Notice
+
+  - The ``collStats`` helper is deprecated. Use the :manual:`$collStats
+    </reference/operator/aggregation/collStats>` aggregation operator instead.
+  - Methods of ``db.command()`` that were incorrectly included in the
+    typescript interface have been deprecated.
```

Comment:
I: I think these were options, not methods.

```suggestion
  - The TypeScript interface passed to the ``db.command()`` method incorrectly
  includes certain options. These options have been deprecated.
```

Reply:
Oops, good catch. Changed